### PR TITLE
Description update

### PR DESCRIPTION
**Pull request by juliaweingart**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json [hunk 1]
```diff
@@ -49,10 +49,10 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceNamespace}/{resourceType}/{resourceName}/providers/Microsoft.WorkloadMonitor/monitors": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{providerName}/{resourceCollectionName}/{resourceName}/providers/Microsoft.WorkloadMonitor/monitors": {
```
Comment (markcowl) [inline specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json:52]: It looks like this is intended to be an extension resource.  What kinds of scopes can this resource be applied at?

File: specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json [hunk 2]
```diff
@@ -328,124 +328,126 @@
...
     },
-    "resourceType": {
+    "resourceCollectionName": {
       "in": "path",
-      "name": "resourceType",
+      "name": "resourceCollectionName",
       "required": true,
       "type": "string",
-      "description": "The resourceType of the resource",
+      "description": "The resource collection name (ex: virtualMachines for virtual machines).",
```
Comment (markcowl) [inline specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json:355]: Is the intention that this only applies to tracked resources, or can it apply to sub-resources as well?  Is there a possibility this value could contain a '/'?

**Review by markcowl — APPROVED** [review]

File: specification/workloadmonitor/resource-manager/readme.md [hunk 3]
```diff
@@ -59,6 +59,14 @@ directive:
 ```
 
 ---
+
+## Suppression:
+``` yaml
+directive:
+  - suppress:
+      - R4017 # HealthMonitors only applied to top-level resources.
```
Comment (markcowl) [inline specification/workloadmonitor/resource-manager/readme.md:67]: We need to scope this suppression: adding a 'from' to indicate the file it is suppressed in is fine.

Here is an example: https://github.com/Azure/azure-rest-api-specs/blob/6455851aa3d2882cb6482e5b1d76c92138b38447/specification/cosmos-db/resource-manager/readme.md

**Review by markcowl — CHANGES_REQUESTED** [review]

File: specification/workloadmonitor/resource-manager/readme.md (hunk 3 above)
Comment (juliaweingart) [reply specification/workloadmonitor/resource-manager/readme.md:68]: Am I doing this wrong? The error is no longer getting suppressed.

**Review by markcowl — APPROVED** [review]

**Pull request merged**